test_projects/cnode/test/example_test.exs

```diff
@@ -104,6 +104,13 @@ defmodule ExampleTest do
     assert {:ok, ^pid} = Unifex.CNode.call(context[:cnode], :test_pid, [pid])
   end
 
+  test "struct", context do
```

Comment:
I would also add test for improper struct to see if error messages are correct etc. We can e.g. use struct with improper field or try to pass other type as struct. Test with recursive struct would also be very useful.

Reply:
Ahh right, `def_struct` has been already taken too. I am not convinced about `struct_def` as it breaks convention (we probably always have `def_xxx`). Maybe `def_c_struct` or `def_unifex_struct`? @mat-hek do you have any ideas?